### PR TITLE
Compress static images.

### PR DESCRIPTION
**Pull request by sgomes**
All images were run through `ImageOptim` with default settings.

The compression applied should be lossless, using improved deflate algorithms and metadata removal.

Saves a total of 1.8MB out of 10.2MB.

#### Changes proposed in this Pull Request

* Compress all images in `static/` using `ImageOptim`.

#### Testing instructions

* Ensure that the various static images, particularly JPEG images, didn't suffer any loss in quality. Loss in quality is not expected, as the tool attempted to preserve quality by performing lossless compression, but since JPEG is a lossy encoding there could be some loss on re-encoding.

### CODE REVIEW

File: static/images/color-schemes/color-scheme-thumbnail-nightfall.svg [hunk 1]
```diff
@@ -1,25 +1 @@
...
-	<path class="st2" d="M0,86h70v16H0V86z"/>
-	<path class="st3" d="M7,91h55v6H7V91z"/>
-	<path class="st3" d="M7,75h37v6H7V75z M7,107h29v6H7V107z M7,123h45v6H7V123z"/>
-	<circle class="st4" cx="35.5" cy="45.5" r="16.5"/>
-	<path class="st3" d="M90,36h130v107H90V36z"/>
-	<path class="st5" d="M163,48h42c1.7,0,3,1.3,3,3v10c0,1.7-1.3,3-3,3h-42c-1.7,0-3-1.3-3-3V51C160,49.3,161.3,48,163,48z"/>
-	<path class="st6" d="M0,0h240v20H0V0z"/>
-</g>
-</svg>
+<svg enable-background="new 0 0 240 160" viewBox="0 0 240 160" xmlns="http://www.w3.org/2000/svg"><g clip-rule="evenodd" fill-rule="evenodd"><path d="m70 20h170v140h-170z" fill="#f6f6f6"/><path d="m0 20h70v140h-70z" fill="#204a69"/><path d="m0 86h70v16h-70z" fill="#7e280e"/><path d="m7 91h55v6h-55z" fill="#fff"/><path d="m7 75h37v6h-37zm0 32h29v6h-29zm0 16h45v6h-45z" fill="#fff"/><circle cx="35.5" cy="45.5" fill="#b0b5b8" r="16.5"/><path d="m90 36h130v107h-130z" fill="#fff"/><path d="m163 48h42c1.7 0 3 1.3 3 3v10c0 1.7-1.3 3-3 3h-42c-1.7 0-3-1.3-3-3v-10c0-1.7 1.3-3 3-3z" fill="#7c589f"/><path d="m0 0h240v20h-240z" fill="#1a1a1a"/></g></svg>
```
Comment (jsnajdr) [inline static/images/color-schemes/color-scheme-thumbnail-nightfall.svg]: I like how this SVG is optimized by inlining the styles 🙂 I verified that the color scheme thumbnails have the right colors on the Account Settings screen :+1:

**Review by jsnajdr — APPROVED** [review]
I didn't verify everything of course, but checking a few selected SVGs and PNGs didn't show any breakage :+1:

**Comment by sgomes** [comment]
Thanks for the review, @jsnajdr !

File: static/images/color-schemes/color-scheme-thumbnail-nightfall.svg (hunk 1 above)
Comment (sgomes) [reply static/images/color-schemes/color-scheme-thumbnail-nightfall.svg]: Yeah, `svgcleaner` seems to do some really clever stuff :)

**Pull request merged**